api/types.proto

```diff
@@ -277,6 +277,50 @@ message UpdateConfig {
 
 	// Amount of time between updates.
 	Duration delay = 2 [(gogoproto.nullable) = false];
+
+	enum FailureAction {
+		PAUSE = 0;
+		CONTINUE = 1;
+		// TODO(aaronl): Add ROLLBACK as a supported failure mode.
+		// (#486)
+	}
+
+	// FailureAction is the action to take when an update failures.
+	// Currently, a failure is defined as a single updated task failing to
+	// reach the RUNNING state. In the future, there will be configuration
+	// to define what is treated as a failure (see #486 for a proposal).
+	FailureAction failure_action = 3;
+}
+
+// UpdateStatus is the status of an update in progress.
+message UpdateStatus {
+	enum UpdateState {
+		NEW = 0;
+		UPDATING = 1;
+		PAUSED = 2;
+		COMPLETED = 3;
```

Comment:
Should we use custom enum names?

I think we should do that (so we avoid names like `UpdateStatus_NEW`) but at the same time we've been inconsistent in the rest of the code so it doesn't really matter